homeTask_1/homeTask_1/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace homeTask_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Console.WriteLine("Hello World!");
+            //Задача- вывести в консоль максимальные и минимальные допустимые значения для
+            //типов данных byte, short, int, long, float, double, uint, ulong.
+            byte bMax = byte.MaxValue;
```

Comment:
It's not necessary to create min/max/name variables for each data type. These values are already stored in System.
Thus, we simply fill the memory with data that already exists. Please refactor it.

Reply:
Hi @EgorAntonovich. No problem. I will fix it.